src/__tests__/index.test.js

```diff
@@ -67,6 +69,35 @@ describe('Get Osmose errors', () => {
   });
 });
 
+describe('Update Osmose errors', async () => {
+  const language = 'fr';
+  const osmose = new OsmoseRequest({ language });
+  let errors = await osmose.fetchErrors({ item: 8120 });
+
+  it('Should close error related to an Osmose item', async () => {
+    const errorId = errors[0].error_id;
+    const result = await osmose.closeError(errorId);
```

Comment:
You forgot to create a mock in the `src/__mocks__/requests.js` file. So you are actually calling the Osmose service and close an error :/